### PR TITLE
Implementation of floating disks attach and detach action

### PR DESCRIPTION
**Pull request by pdilung**
This pull request implements attach and detach actions of floating disks as described in #60.

### CODE REVIEW

File: lib/rbovirt.rb [hunk 1]
```diff
@@ -131,10 +133,11 @@ def http_put(suburl, body, headers={})
       end
     end
 
-    def http_delete(suburl)
+    def http_delete(suburl, body, headers={})
```
Comment (abenari) [inline lib/rbovirt.rb]: By changing the method to 
```ruby
def http_delete(suburl, body=nil, headers={})
```
You will not need to use nil each time you call the method.

Comment (pdilung) [inline lib/rbovirt.rb]: Ok, let me update the method definition as well as its calls.

**Comment by abenari** [comment]
@abonas @jhernand care to review this patch? I find it useful and would like to merge it if no objections on your side.

**Comment by jhernand** [comment]
I don't have any objection, looks good.

**Pull request merged**